yarn-error.log

```diff
@@ -6605,14 +6605,11 @@ Lockfile:
     resolved "https://registry.yarnpkg.com/zone.js/-/zone.js-0.8.20.tgz#a218c48db09464b19ff6fc8f0d4bb5b1046e185d"
 
 Trace: 
-  Error: Outdated lockfile. Please run `yarn install` and try again.
+  Error: Couldn't find package "coverage-istanbul" on the "npm" registry.
       at new MessageError (/Users/bo/.nvm/versions/node/v8.10.0/lib/node_modules/yarn/lib/cli.js:186:110)
-      at /Users/bo/.nvm/versions/node/v8.10.0/lib/node_modules/yarn/lib/cli.js:36620:19
+      at NpmResolver.<anonymous> (/Users/bo/.nvm/versions/node/v8.10.0/lib/node_modules/yarn/lib/cli.js:51625:15)
       at Generator.next (<anonymous>)
       at step (/Users/bo/.nvm/versions/node/v8.10.0/lib/node_modules/yarn/lib/cli.js:98:30)
-      at /Users/bo/.nvm/versions/node/v8.10.0/lib/node_modules/yarn/lib/cli.js:116:14
-      at new Promise (<anonymous>)
-      at new F (/Users/bo/.nvm/versions/node/v8.10.0/lib/node_modules/yarn/lib/cli.js:23451:28)
-      at /Users/bo/.nvm/versions/node/v8.10.0/lib/node_modules/yarn/lib/cli.js:95:12
-      at /Users/bo/.nvm/versions/node/v8.10.0/lib/node_modules/yarn/lib/cli.js:36660:25
-      at Array.map (<anonymous>)
+      at /Users/bo/.nvm/versions/node/v8.10.0/lib/node_modules/yarn/lib/cli.js:109:13
+      at <anonymous>
+      at process._tickCallback (internal/process/next_tick.js:188:7)
```

Comment:
maybe we should include the whole 'yarn-error.log' in the .gitignore file?

Reply:
good idea, will do!